.gitignore

```diff
@@ -14,7 +14,7 @@ dist/
 downloads/
 eggs/
 .eggs/
-lib/
+./lib/
```

Comment:
This is probably not what was intended here...but also `lib` is a typical code directory. Is there something more pythonic I can call this dir? `pkg`?

Reply:
i don't know maybe `utils`?